staubli_rx160_gazebo/urdf/rx160.xacro

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<robot name="staubli_rx160_gazebo" xmlns:xacro="http://ros.org/wiki/xacro">
```

Comment:
We could standardise on adding the `gazebo` suffix to all Gazebo compatible xacros, but there is also something to say for just keeping it the same as the normal model name (ie: `staubli_rx160` in this case).

Personally I'd prefer to keep the original name. Did you have any particular reason to append `_gazebo` (name clashes, something not working, personal preference, etc)?